packages/ketcher-core/src/application/render/restruct/remultitailArrow.ts

```diff
@@ -46,6 +48,10 @@ export class ReMultitailArrow extends ReObject {
     super(MULTITAIL_ARROW_KEY);
   }
 
+  getOffset(options: RenderOptions) {
```

Comment:
Can we rename it to getFrameOffset?

Reply:
And it would be great if it is possible to name constant numbers somehow to describe them.